addons/website_sale/controllers/delivery.py

```diff
@@ -187,6 +187,55 @@ def _get_rate(carrier, order, is_express_checkout_flow=False):
                     rate['price'] = taxes['total_included']
         return rate
 
+    def order_lines_2_google_api(self, order_lines):
+        """ Transforms a list of order lines into a dict for google analytics """
+        order_lines_not_delivery = order_lines.filtered(lambda line: not line.is_delivery)
```

Comment:
Useless, the filter is already applied in main controller

Reply:
Sorry, didn't notice that was kept during the rebase :/ Same for all duplicated code aie aie